src/lilv_internal.h

```diff
@@ -34,13 +34,18 @@ extern "C" {
 #include <stdlib.h>
 
 #ifdef _WIN32
+#    undef _WIN32_WINNT
+/** Force XP/2003 for excluding CreateSymbolicLinkA/W **/
+#    define _WIN32_WINNT 0x0500
```

Comment:
Redefining system version constants is extremely questionable and could have any number of effects aside from this.  This needs to be done another way.